static/elements/chromedash-featurelist.js

```diff
@@ -199,7 +199,7 @@ class ChromedashFeaturelist extends LitElement {
   }
 
   _filterKeyword(regExp, feature) {
-    return (feature.name + '\n' + feature.summary + '\n' + feature.comments)
+    return (feature.name + '\n' + feature.summary + '\n' + feature.comments+ '\n' + feature.keyword)
```

Comment:
In models.py this search tag information is added to the JSON representation of the feature under the name "tags", so you need to change "keyword" to "tags".

Also, almost all our code is formatted to wrap at 80 columns, so please word-wrap this line.

Reply:
Done.